Watchman.Discord/Areas/Responses/BotCommands/ResponseCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Framework.Commands.PropertyAttributes;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+
+namespace Watchman.Discord.Areas.Responses.BotCommands
+{
+    public class ResponsesCommand : IBotCommand
+    {
+        [Text]
+        public string Type { get; set; }
```

Comment:
nie lepiej boolami jak przy -stats
?

Reply:
wcześniej gdy miałem z boolami to miałem pewien błąd, który wynikał z źle napisanego kodu w innym miejscu, ale obecnie już tego problemu nie ma, więc mogę zrobić z boolami.

Chociaż jak tak sobie myślę, to może lepiej to zostawić jako text, ponieważ wtedy komenda jest bardziej uniwersalna i wygodniejsza w użyciu dla usera. Dzięki temu można napisać "-responses -custom" lub "-responses custom" i obie wersje są poprawne i jeszcze w dodatku wielkość znaków nie ma znaczenia. Nawet powiem, że chyba też tak powinno być dla -stats bo też będzie można używać ją w podobny sposób co -responses. Co o tym myślisz?